skimage/transform/_geometric.py

```diff
@@ -58,6 +58,8 @@ def _center_and_normalize_points(points):
            (1997): 580-593.
 
     """
+    # deal with array_like points
+    points = np.array(points)
```

Comment:
I think we can use `np.asarray`, which only copies if necessary. In contrast, I believe `np.array` always makes a copy. This could be a performance regression otherwise.